src/plat/rk3288/overlay-rk3288-ntablet.dts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: GPL-2.0-only
+ */
+
+/ {
+	chosen {
+		seL4,elfloader-devices =
+		    "serial2";
+		seL4,kernel-devices =
+		    "serial2",
+		    &{/interrupt-controller@ffc01000},
+		    &{/timer};
+	};
+
+	// Trustzone carves out some RAM for its secure zone.
+	// Ideally we'd use the memory node passed in from U-Boot,
```

Comment:
The kernel isn't allowed to use any DTS parsing code during runtime (unless we verify it), so we will always need to statically know what the memory node is at compile time.

Reply:
unfortunately we don't know necessarily which region is carved out.    Reflashing OP-TEE leads to a different hole...  flashing a different u-boot version leads to a different hole.  So it's hard to add a memory node that is correct.

Reply:
Do you know the rational for changing the position of the hold for different versions ... Or can we specify a region that is big enough to cover all holes for different versions ...

Reply:
Depends on which version you use.  
The hole I put in here is for the OPTEE Rockchip provides.  When I try to build one it wants a different hole.

Reply:
We have a similar issue with RISC-V and OpenSBI, but the carved out memory is always the same there. One way to avoid adding code to the kernel would be handling this in the ELF Loader (assuming it exists, https://github.com/seL4/seL4_tools/issues/108) and then we could have a custom boot interface (https://sel4.discourse.group/t/pre-rfc-arm-boot-interface-versioning/410/4) to pass the essence.